core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPushProjectionIntoTableScan.java

```diff
@@ -148,6 +148,10 @@ public void testPushProjection()
                     identity, "projected_variable_" + connectorNames.get(identity),
                     dereference, "projected_dereference_" + connectorNames.get(dereference),
                     constant, "projected_constant_" + connectorNames.get(constant));
+            ImmutableMap<String, ColumnHandle> expectedColumns = newNames.entrySet().stream()
```

Comment:
declare as `Map`

Reply:
i was trying to be consistent with previous variable declarations here

Reply:
If we are declaring variables as `ImmutableMap/List/Set` then this is an ommision. Actually we should probably have a checkstyle rule for that.

Reply:
Changed mine to `Map`